### PR TITLE
At-rule functions

### PR DESCRIPTION
**Pull request by ramiy**
### Description

A page with all the available CSS [at-rules](https://developer.mozilla.org/en-US/docs/Web/CSS/At-rule) functions.

### Motivation

CSS data-types have functions. They are documented in https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions

CSS at-rule have functions too, they are not documented yet. This PR adds a new page containing those functions.

### Additional details

Back in 2020, when we used the old platform before the migration to GitHub markdown, [I created the first page](https://github.com/mdn/sprints/issues/3838) listing all the available "[CSS functions](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions)". It contains dozens of data-type-functions in one place.

In 2023, I want to create the same page with all the available at-rule-functions in one place. Hopefully, spec writers will add new at-rule-functions in the future.

### CODE REVIEW

**Comment by dipikabh** [comment]
Hi @ramiy, I've started reviewing your pull request. I have a couple of questions/observations related to the other page https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions:

- Should the title "CSS Functional Notation" instead be "CSS data type functions" or "CSS type functions" to be more explicit?
- Not all the functions listed on this page appear in the sidebar under the "Functions" menu. Shouldn't they?
- The landing pages - both for type functions and at-rule functions - should themselves be accessible from the sidebar. The landing page for the data types https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Types also cannot be currently reached from the side bar.

File: files/en-us/web/css/at-rule-functions/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## @supports functions
+
+The {{CSSxRef("@supports")}} at-role specify CSS declarations that depend on a browser's support for CSS features.
+
+- {{CSSxRef("@supports", "@supports selector()")}}
+  - : Evaluates browser supports based on specified selector syntax.
+- {{CSSxRef("@supports", "@supports font-tech()")}}
+  - : Evaluates browser supports based on specified font technology for layout and rendering.
+- {{CSSxRef("@supports", "@supports font-format()")}}
+  - : Evaluates browser supports based on specified font format for layout and rendering.
```
Comment (dipikabh) [inline files/en-us/web/css/at-rule-functions/index.md:49]:
```suggestion
The {{CSSxRef("@supports")}} at-rule checks for a browser's support for the specified CSS feature and then applies the CSS styling.

- {{CSSxRef("@supports", "@supports selector()")}}
  - : Applies CSS rules after checking browser's support for the specified selector syntax.
- {{CSSxRef("@supports", "@supports font-tech()")}}
  - : Applies CSS rules after checking browser's support for the specified font technology.
- {{CSSxRef("@supports", "@supports font-format()")}}
  - : Applies CSS rules after checking browser's support for the specified font format.
```

File: files/en-us/web/css/at-rule-functions/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## @namespace functions
+
+The {{CSSxRef("@namespace")}} at-role specify XML namespaces to be used in a CSS stylesheet.
+
+- {{CSSxRef("@namespace", "@namespace url()")}}
+  - : Defines XML namespace from a specified URL.
+- {{CSSxRef("@namespace", "@namespace src()")}}
+  - : Defines XML namespace from a specified source.
```
Comment (dipikabh) [inline files/en-us/web/css/at-rule-functions/index.md:58]:
```suggestion
The {{CSSxRef("@namespace")}} at-rule is used to specify XML namespaces to be used in a CSS stylesheet.

- {{CSSxRef("@namespace", "@namespace url()")}}
  - : Defines XML namespace from the specified URL.
- {{CSSxRef("@namespace", "@namespace src()")}}
  - : Defines XML namespace from the specified source.
```

File: files/en-us/web/css/at-rule-functions/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## @container functions
+
+The {{CSSxRef("@container")}} at-role specify styles to a containment context.
```
Comment (dipikabh) [inline files/en-us/web/css/at-rule-functions/index.md:62]:
```suggestion
The {{CSSxRef("@container")}} at-rule is used to specify styles for a containment context.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @ramiy, I've added a few edits and suggestions.

File: files/en-us/web/css/at-rule-functions/index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+title: CSS at-rule functions
+slug: Web/CSS/At-rule-functions
```
[the remark this answers is not in the document]
Comment (ramiy) [reply files/en-us/web/css/at-rule-functions/index.md:3]: Tried to use "At-rule_functions" but got an error... I reverted to "At-rule-functions"

**Review by dipikabh — APPROVED** [review]
Thanks for the updates.

**Comment by dipikabh** [comment]
I've left a comment about the slug change.

Any thoughts about this: https://github.com/mdn/content/pull/23523#issuecomment-1382412991?

**Comment by ramiy** [comment]
@dipikabh I'm currently updating the CSS functions page. When I create the PR, I'll ping you. We will discuses about all the issues in a separate PR. This PR is all about at-rule functions...

**Pull request merged**